x-pack/solutions/observability/plugins/streams_app/public/components/stream_detail_enrichment/processors/grok/index.tsx

```diff
@@ -20,6 +20,7 @@ export const GrokProcessorForm = () => {
       <ProcessorFieldSelector />
       <GrokPatternsEditor />
       <EuiSpacer size="m" />
+      <EuiSpacer size="m" />
```

Comment:
Probably you don't need a new spacer but just update the size of the one already present above